### ISSUE TITLE
Learn WordPress Content Feedback - error regarding WordCamp organizer tickets in quiz

### ISSUE THREAD

**Issue by kaitohm** [WordPress/Learn#1655]
# Feedback Details
- Content title: Global Community Sponsorship — For Event Organizers Quiz
- Content URL: https://learn.wordpress.org/quiz/global-community-sponsorship-for-event-organizers-2-3/
- Are you reporting an error (such as outdated information), or providing improvement ideas?: error

**Write a description of your feedback here:**

Handbook says free tickets are given to all levels of global sponsorship. Question in this quiz says that they are NOT. I think. Unless I'm going crazy from reading all of this right now. But I'm pretty sure there is error in the quiz.

Handbook: https://make.wordpress.org/community/handbook/wordcamp-organizer/planning-details/fundraising/global-community-sponsorship-for-event-organizers/

_This feedback was submitted in ticket 160449._

---

# Feedback Report Checklist:
- [ ] Feedback validated
- [ ] Update proposed
- [ ] Update reviewed
- [ ] Update published

**Comment by kaitohm** [comment]
Asked for a review by the Community Team in Slack: https://wordpress.slack.com/archives/C037W5S7X/p1687841466111349

**Comment by nukaga** [comment]
Yes. Bronze sponsors receive "A number of free tickets to each in-person WordCamp, reserved for your company representatives."
Therefore, it is incorrect to answer "NOT included" in the quiz with "Several free tickets to each WordCamp".

I'm sorry. I don't seem to have the authority to modify this quiz. I don't understand how it works, should I ask the person who created this quiz on the community team to fix it?

**Comment by kaitohm** [comment]
@nukaga I can make the changes for you 👍 

I have updated the quiz to the following. Does this look correct?
<img width="926" alt="Screenshot 2023-07-08 at 9 31 03" src="https://github.com/WordPress/Learn/assets/49054298/7088afbc-2cc5-403f-a46f-10076e4b13b5">
